tika-core/src/main/java/org/apache/tika/parser/RecursiveParserWrapper.java

```diff
@@ -145,7 +145,7 @@ public void parse(InputStream stream, ContentHandler recursiveParserWrapperHandl
         int writeLimit = -1;
         //TODO -- rely on a new interface WriteLimiting...?
         //It'd be better not to tie this to a specific class
-        if (recursiveParserWrapperHandler instanceof BasicContentHandlerFactory) {
+        if (recursiveParserWrapperHandler instanceof BasicContentHandlerFactory) { // TODO this cond is always false
```

Comment:
Do you want to open a JIRA ticket if this is a problem?

Reply:
https://issues.apache.org/jira/browse/TIKA-3652